src/com/pentalog/pentastagiu/homework2_bookCandyShop/MainClass.java

```diff
@@ -0,0 +1,91 @@
+package com.pentalog.pentastagiu.homework2_bookCandyShop;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class MainClass {
```

Comment:
este ok cum ai testat, e bine ca ai adaugat niste cazuri mai complicate si se vede ca ai investit timp pentru tema
ti-as sugera tot asa, cand ai timp, sa incerci sa refactorizezi putin astfel incat sa fie mai usor de citit codul

o sa vezi ca e foarte important asta, pt ca scrii codul o data, dar e citit de vreo 10 ori; si astfel daca citeste cineva codul tau sau daca tu o sa vrei sa il citesti peste 3 luni, sa citesti linie cu linie si sa fie f clar ce se intampla is usor de urmarit logica